com.github.sqlitebrowser.sqlitebrowser.yaml

```
@@ -0,0 +1,22 @@
+app-id: com.github.sqlitebrowser.sqlitebrowser
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: sqlitebrowser
+rename-icon: sqlitebrowser
+rename-appdata-file: sqlitebrowser.appdata.xml  
```

```suggestion
rename-appdata-file: sqlitebrowser.desktop.appdata.xml
```
That's the correct file name upstream, you probably want to rename those upstream